src/main.nr

```diff
@@ -7,7 +9,16 @@ fn main(value: u32, hash_value: pub u32){
 }
 
 fn calculate_hash(num: u32) -> u32 {
-    
+    let size = 2.pow_32(32) as u64;
```

Comment:
Since this is a fixed value best to avoid calculating it here. Instead declare it as a global or const